module/VuFind/config/module.config.php

```diff
@@ -830,6 +831,41 @@
 $routeGenerator->addDynamicRoutes($config, $dynamicRoutes);
 $routeGenerator->addStaticRoutes($config, $staticRoutes);
 
+// API routes
+$config['router']['routes']['searchApiHome'] = [
```

Comment:
Perhaps it would make sense at some point to add an "addApiRoute" method to the route generator to reduce the verbosity of this configuration, especially if we expect over time to be adding a lot of variations with different version numbers.